scripts/terminal.py

```diff
@@ -22,10 +22,13 @@
 SOFTWARE.
 """
 import _lines_of_code
+from github import Github
 
 
 token = input('Please enter you token: ')
-user_id = input('Please enter an user id (Leave blank to scan your own account): ')
+user_name = input('Input a Github user\'s name (or leave it blank for your own account): ')
+g = Github(token)
+user_id = g.get_user(user_name).id
 if user_id:
     user_id = int(user_id)
```

Comment:
No need to find user id while we can get the user with username alone. ```github.get_user(username)``` can be used in scripts/_lines_of_code.py